pkg/chart/loader/load.go

```diff
@@ -205,3 +206,45 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 
 	return c, nil
 }
+
+func LoadValues(data []byte) (map[string]interface{}, error) {
```

Comment:
```suggestion
func LoadValues(data io.Reader) (map[string]interface{}, error) {
```

It is preferred to take an interface that can support the necessary functionality, rather than a concrete type